src/GitVersionCore/Configuration/BranchConfig.cs

```diff
@@ -17,42 +17,72 @@ public BranchConfig(BranchConfig branchConfiguration)
             TagNumberPattern = branchConfiguration.TagNumberPattern;
             TrackMergeTarget = branchConfiguration.TrackMergeTarget;
             CommitMessageIncrementing = branchConfiguration.CommitMessageIncrementing;
-            IsDevelop = branchConfiguration.IsDevelop;
+            TracksReleaseBranches = branchConfiguration.TracksReleaseBranches;
             IsReleaseBranch = branchConfiguration.IsReleaseBranch;
             IsMainline = branchConfiguration.IsMainline;
         }
 
         [YamlMember(Alias = "mode")]
-        public VersioningMode? VersioningMode { get; set; }
+        public VersioningMode? VersioningMode
```

Comment:
@maxraab If the current ReSharper settings made the properties multiline, we should indeed fix that in its own PR.